### PR TITLE
Add logo

### LINKED ISSUE #3: Logo
I think a logo for this community would be nice.
With my extremely limited GIMP fu I came up with this:
rust-embedded-community-logo

### PR DESCRIPTION
**Pull request by eldruin**
[Rendered](https://github.com/eldruin/rust-embedded-community-meta/tree/add-logo)
Blocked on https://github.com/rust-embedded/wg/issues/482

Closes #3

### CODE REVIEW

File: assets/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+[Legal Policy]: https://www.rust-lang.org/id-ID/legal.html
+[Art]: https://www.rust-lang.org/id-ID/legal.html#art
+[Trademark Policy]: https://www.rust-lang.org/id-ID/legal.html#trademark-policy
+
+## Attribution
+
+### Logo
+
+The Rust Embedded Community Logo was designed by [Diego Barrios Romero]. It is based on the Rust Embedded Working Group Logo, based on the Rust logo, which was designed by [Aaron Power]. These designs include all files matching `assets/logo/rec-logo*`.
```
Comment (jamesmunns) [inline assets/README.md:15]:
```suggestion
The Rust Embedded Community Logo was designed by [Diego Barrios Romero]. It is based on the Rust Embedded Working Group Logo, based on the Rust logo, which was designed by [Erin Power]. These designs include all files matching `assets/logo/rec-logo*`.
```

File: assets/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+[Aaron Power]: https://github.com/Aaronepower
```
Comment (jamesmunns) [inline assets/README.md:17]:
```suggestion
[Erin Power]: https://github.com/XAMPPRocky
```

**Comment by jamesmunns** [comment]
Added comments to correct Erin's name and GH username

**Comment by eldruin** [comment]
Closing in favour of #9

**Pull request closed without merging**